src/app/metadata/features/dataset-details/dataset-details.component.html

```diff
@@ -462,7 +462,20 @@ <h4 class="inline-flex items-center">
                   File hash
                 </th>
                 <td mat-cell *matCellDef="let file">
-                  {{ file.file_information?.sha256_hash }}
+                  @if (file.file_information?.sha256_hash) {
+                    <span
+                      class="inline-block max-w-[8ex] overflow-hidden text-ellipsis whitespace-nowrap"
```

Comment:
```suggestion
                      class="inline-block max-w-[10ex] overflow-hidden text-ellipsis whitespace-nowrap"
```

This more or less reliably shows 7 significant digits (more or less because the font is not monospaced, but I think it's good enough).